examples/structured_data/advanced_techniques_for_structured_data.py

```diff
@@ -252,90 +252,53 @@ def create_model_inputs():
 
 
 """
-## Create feature columns
+## Encode features
 
 We create two representations of our input features: sparse and dense:
-1. In the **sparse** representation, the categorical features are encoded with one-hot encoding,
-using [tf.feature_columns.indicator_column](https://www.tensorflow.org/api_docs/python/tf/feature_column/indicator_column).
-This representation can be useful for the model to *memorize* particular feature values to make certain
-predictions.
-2. In the **dense** representation, the categorical features are encoded with low-dimensional embeddings,
-using the [tf.feature_column.embedding_column](https://www.tensorflow.org/api_docs/python/tf/feature_column/embedding_column).
+1. In the **sparse** representation, the categorical features are encoded with one-hot encoding, using `CategoryEncoding` layer. 
```

Comment:
For consistency and grammar, use "the":

```suggestion
1. In the **sparse** representation, the categorical features are encoded with one-hot encoding using the `CategoryEncoding` layer. 
```

(similar to what you wrote later: `...dimensional embeddings, using the `Embedding` layer...`)